### PR TITLE
DEV: add logging and configuration info for Redis Insight

### PR DESCRIPTION
**Pull request by dwdougherty**
Ticket: [DOC-3757](https://redislabs.atlassian.net/browse/DOC-3757)

Staged previews:

- https://redis.io/docs/staging/DOC-3757/develop/connect/insight/#log-files
- https://redis.io/docs/staging/DOC-3757/operate/redisinsight/install/configuration/

### CODE REVIEW

File: content/operate/redisinsight/install/configuration.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+To configure Redis Insight with environment variables:
+
+1. Follow the instructions to set environment variables for your operating system:
+
+    - [macOS](https://apple.stackexchange.com/a/106814)
+    - [Windows](https://www.architectryan.com/2018/08/31/how-to-change-environment-variables-on-windows-10/)
+    - [Linux](https://askubuntu.com/a/58828)
+    - [Docker](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file)
+
+1. Set the environment variables.
```
Comment (andy-stark-redis) [inline content/operate/redisinsight/install/configuration.md:19]: Do we need this line?

File: content/operate/redisinsight/install/configuration.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+1. Restart Redis Insight.
+
+## Redis Insight environment variables
+
+The following environment variables can be set to configure Redis Insight:
```
Comment (andy-stark-redis) [inline content/operate/redisinsight/install/configuration.md:24]: Maybe "Use the following environment variables to configure Redis Insight" to avoid PV?

**Review by andy-stark-redis — APPROVED** [review]
Few minor things, maybe, but otherwise language looks fine (but I've no idea about the actual info).

File: content/operate/redisinsight/install/configuration.md (hunk 1 above)
Comment (dwdougherty) [reply content/operate/redisinsight/install/configuration.md:19]: Haha... no. :) I'll remove it next push.

File: content/operate/redisinsight/install/configuration.md (hunk 2 above)
Comment (dwdougherty) [reply content/operate/redisinsight/install/configuration.md:24]: Good catch! Thank you.

**Comment by dwdougherty** [comment]
Thank you, @andy-stark-redis !

**Pull request merged**